data/routing.txt

```diff
@@ -15,8 +14,15 @@ Bitcoin         | Coinbase   | 1% + 15¢     | On request |
 
 Route           | Provider   | Fee          | When       | Details
 ----------------+------------+--------------+------------+-----------
-Bank Account    | Balanced   | 0            | At payday  | U.S. only
```

Comment:
> So, the `In` route was not through the ACH, right?

Correct. We've never offered bank payins (#777).

> The graph becomes more complicated - need to make clear that Credit Card belongs to Bank and Bank and Credit Card didn't change during the transition. Then the money from the Bank are being transferred by Balanced or Braintree (how to call them?) for the In direction.
>
And then money go into the Bank, they move first to ACH by Zipmark and then to Card of the Bank. Is that right?

I don't understand this. :-(

What do you mean by "Credit Card belongs to Bank"?